changes/ee/feat-13504.en.md

```diff
@@ -0,0 +1 @@
+Added a HTTP backend for the authentication mechanism `scram`.
```

Comment:
please briefly describe how it works.
should also emphasize that this is not rfc7804